### ISSUE TITLE
Unable to login to Steam3

### ISSUE THREAD

**Issue by PantACRO4life** [SteamRE/DepotDownloader#342]
Ok so First I want to say that I read all the guide and I still have no succes
Version: 2.4.6

![image](https://user-images.githubusercontent.com/6643071/181080127-86082866-c7e4-4a91-bba2-344ff51a9a91.png)

I try with a less complex password, with and without steam guard, I always have the same result.

Please Help, thanks

**Comment by PantACRO4life** [comment]
![image](https://user-images.githubusercontent.com/6643071/181081679-f3cf8302-6d69-4a61-a3f9-f73cb83979a6.png)

**Comment by PantACRO4life** [comment]
I went to another steam account, purchase the game I wanted to downgrade, did the exact same thing as above, note that both account had the same exact password at this point of my test, and with this other steam account that procedure worked.

I still would like to have it work on my main account because there is other game I need to downgrade and I am not going to buy them all......

**Comment by azuisleet** [comment]
This error message is very specific in that you have provided the wrong username or password. If you are certain about your password, you have supplied the wrong username.

**Comment by cebdos** [comment]
hello, I have the same issue even when disabling the firewall, my IDs are correct

I get this entry
![image](https://user-images.githubusercontent.com/110831527/183428859-b2587a44-d0b8-4d3b-9912-324006625740.png)

**Comment by yaakov-h** [comment]
You've been rate-limited for one reason or another. If you're spamming Steam or running a large number of instances, then don't.

**Comment by cebdos** [comment]
yes sorry it happened after I got InvalidPassword many times, the original error was the following one
![image](https://user-images.githubusercontent.com/110831527/183448891-917cb882-4c6a-41f5-a63d-46928b01ba3d.png)

**Comment by cebdos** [comment]
I found the issue, I changed my password and got rid of special characters in it, now it works

**Comment by PantACRO4life** [comment]
> This error message is very specific in that you have provided the wrong username or password. If you are certain about your password, you have supplied the wrong username.

I 10000% sure of the username for both account and have both the same password without any special characters.

1 account work while the other one doesnt behing the main account.

**Comment by cebdos** [comment]
do you have special character in the username of your account that doesn't work maybe?

**Comment by PantACRO4life** [comment]
> do you have special character in the username of your account that doesn't work maybe?

no only letter and number for both account

**Comment by cebdos** [comment]
ok then I guess it's a different issue than mine
